libqtile/backend/base.py

```diff
@@ -212,10 +213,6 @@ def info(self) -> Dict[str, Any]:
         """Return information on this window."""
         return {}
 
-    def cmd_info(self) -> Dict:
-        """Return a dictionary of info."""
-        return self.info()
-
 
```

Comment:
We are removing `cmd_info` but keeping `info` here. In the case where a command was required by a base class, does the abstract method need decorating, or the implemention methods, or both?

Reply:
If I've understood correctly the abstract method doesn't need it; the `expose` decorator is strictly providing functionality for the IPC. I wonder if we can make it so that any implementations of cmd_ abstract methods are necessarily exposed, otherwise the classes run the risk of not always matching each other. This *is* ensured by using `cmd_` prefix. Perhaps this could be accomplished by keep track of exposed methods on the object, rather than the methods themselves. I'm just investigating this.

Reply:
I think that's correct. The decorator is only relevant for the method that is ultimately exposed by the class. If a subclass has a decorated method but this is overriden by an undecorated method then the command will no longer be exposed.

Your idea sounds good.